etc/adminhtml/system.xml

```diff
@@ -156,7 +156,7 @@
                 <field id="enable_order_intent" translate="label" type="select" sortOrder="75" showInDefault="1"
                        showInWebsite="1" showInStore="1">
                     <label>Enable order intent</label>
-                    <comment>Perform a check during checkout to inform buyer whether an order would be accepted by Two before they place order (recommended).</comment>
+                    <comment>Perform a check during checkout to inform buyer whether an order would be accepted before they place order (recommended).</comment>
```

Comment:
suggested => "Perform a check during checkout to inform the buyer whether an order is likely to be accepted before being placed (recommended)."

Reply:
Done